src/status_im2/contexts/onboarding/common/background/view.cljs

```diff
@@ -1,23 +1,56 @@
 (ns status-im2.contexts.onboarding.common.background.view
   (:require [react-native.core :as rn]
             [react-native.blur :as blur]
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [status-im2.common.resources :as resources]
+            [reagent.core :as reagent]
             [status-im2.contexts.onboarding.common.carousel.view :as carousel]
             [status-im2.contexts.onboarding.common.background.style :as style]
             [status-im2.contexts.onboarding.common.carousel.animation :as carousel.animation]))
 
+(def header-text
+  [{:text     (i18n/label :t/join-decentralised-communities)
+    :sub-text (i18n/label :t/participate-in-the-metaverse)}
+   {:text     (i18n/label :t/chat-with-friends)
+    :sub-text (i18n/label :t/with-full-encryption)}
+   {:text     (i18n/label :t/own-your-crypto)
+    :sub-text (i18n/label :t/use-the-multichain-wallet)}
+   {:text     (i18n/label :t/discover-web3)
+    :sub-text (i18n/label :t/explore-the-decentralized-web)}])
+
+(defn background-image
+  [content-width]
+  [rn/image
+   {:style  {:resize-mode :stretch
+             :width       content-width}
+    :source (resources/get-image :onboarding-illustration)}])
+
 (defn view
   [dark-overlay?]
-  [:f>
-   (fn []
-     (let [animate? (not dark-overlay?)]
-       (when animate? (carousel.animation/initialize-animation))
-       [rn/view
-        {:style style/background-container}
-        [carousel/view animate?]
-        (when dark-overlay?
-          [blur/view
-           {:style         style/background-blur-overlay
-            :blur-amount   30
-            :blur-radius   25
-            :blur-type     :transparent
-            :overlay-color :transparent}])]))])
+  (let [view-id  (rf/sub [:view-id])
+        progress (reagent/atom nil)
+        paused   (reagent/atom nil)]
```

Comment:
Hi @J-Son89, Having view-id, progress and paused in background file itself will not work.
1. Carousel will not keep its animated state, on navigation it will reinitialize. I just tried PR and on going back from "I am new to status" screen animations starts from 0
2. Screen with dark-overlay should not be animated [#15534](https://github.com/status-im/status-mobile/issues/15534)
3. Even though other screens have dark-overlay and not animated, they should be at same progress as parent.

Reply:
Yep, a slight oversight in my initial commit. It should be addressed now. 👍